jekyll/_cci2/gh-bb-integration.md

```diff
@@ -286,5 +291,26 @@ jobs:
             - "SO:ME:FIN:G:ER:PR:IN:T"
 ```
 
-When you push to your Bitbucket project from a job,
-CircleCI will use the SSH key you added.
+### How are these keys used?
+
+When CircleCI build your project, we install the private key into the .ssh directory, and configure SSH to use it when communicating with your version control provider. Therefore, it gets used for:
```

Comment:
build -> builds
we-> the private key is installed...and SSH is configured...
.ssh -> `.ssh`